fedimint-ui/apps/guardian-ui/src/components/SetConfiguration.tsx

```diff
@@ -160,7 +159,7 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
             isDisabled={!!statePassword}
           />
           <FormHelperText>
-            {'You\'ll need this every time you visit this page.'}
+            You&#39;ll need this every time you visit this page.
```

Comment:
No issue, but curious about this change? Why?

Reply:
running `yarn format` on the old string with escaped `\'` would change the string to double quote in order to avoid having the escape.

now fixed by enhancing the eslint `quotes` rule definition. See `eslint-config/index.js`